parser/preprocessed/SqlLexer_gen.cpp

```diff
@@ -1300,11 +1280,16 @@ extern int quickstep_yylex \
  */
 YY_DECL
 {
-	yy_state_type yy_current_state;
-	char *yy_cp, *yy_bp;
-	int yy_act;
+	register yy_state_type yy_current_state;
```

Comment:
@hbdeshmukh 
The register variables are generated in my my machine as well ans I get a compilation error. However it seems to have worked for you. Not sure if this is due to the version of flex being used.
We can generate the modified   preprocessed files on my machine and  then commit them